source/connect/connection-options.txt

```diff
@@ -0,0 +1,107 @@
+.. _pymongo-connection-options:
+
+======================
+Set Connection Options
+======================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, configure
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This section describes the MongoDB connection and authentication options
+available in {+driver-short+}. You can configure your connection using either
+the connection URI or arguments to the ``MongoClient`` constructor.
+
+.. _pymongo-connection-uri:
+
+Using the Connection URI
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you pass a connection URI to the ``MongoClient`` constructor, you can include
+connection options in the string as ``<name>=<value>`` pairs. In the following example,
+the connection URI contains the ``connectTimeoutMS`` option with a value of ``60000``
+and the ``tls`` option with a value of ``true``:
```

Comment:
```suggestion
the connection URI contains the ``connect_timeout`` option with a value of ``60000``
and the ``tls`` option with a value of ``True``:
```

Reply:
good call on option name, but "true" is fine for the connection URI